README.md

```diff
@@ -100,19 +100,64 @@ This application creates the following API endpoints:
 
 🔠
 **Words**
+___
+- Returns 4 word choices when no param is specified as default
+- Returns photo url and account info for photo owner to provide photo credits 
+- Returns a correct answer word associated with the photo, 4 word choices (includes answer word)
```

Comment:
These descriptions capture what we need to convey, but they are not reading clearly to me. 

Can it be consolidated to one line... something like: 
Returns a correct answer, 4 words (1 of which is the correct answer) and a photo related to the correct answer

Reply:
Nice word play!!